launcher/src/main/scripts/bin/launcher.py

```diff
@@ -268,9 +274,12 @@ def start(process, options):
     pid = os.fork()
     if pid > 0:
         process.write_pid(pid)
-        print 'Started as %s' % pid
+        print('Started as %s' % pid)
         return
 
+    if need_set_inheritable():
+        os.set_inheritable(process.pid_file.fileno(), True)
```

Comment:
Instead of checking for the version, what if we just always call it and catch/ignore AttributeError?